index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css" />
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+  <title>Cedrich's Portfolio</title>
+</head>
+<body>
+
+  <header class="header">
+    <span class="navItem">
+      <a class="header-text" href="#"><h2>Welcome</h2></a>
+    </span>
+    
+    <nav class="navItem">
+      <a href="#"><i class="fa fa-bars"></i></a>
+    </nav>
+  </header>
+  
+  <main>
+    <section class="content-container">
```

Comment:
- [ ] Kindly remove the main tag, since we just need a section below the header for now :recycle: